packages/core/test/json.test.ts

```diff
@@ -0,0 +1,95 @@
+import { createMachine, assign } from '../src/index';
+import * as machineSchema from '../src/machine.schema.json';
+
+import * as Ajv from 'ajv';
+
+const ajv = new Ajv();
+const validate = ajv.compile(machineSchema);
+
+describe('json', () => {
+  it('should serialize the machine', () => {
```

Comment:
would be good to also add a basic "failing" test to have a confirmation that validation can actually throw